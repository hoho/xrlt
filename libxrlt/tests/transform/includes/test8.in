id:0, type:0, last:0, error:0, data:
id:0, type:0, last:0, error:0, data:
id:0, type:0, last:0, error:0, data:
id:1, type:400, last:1, error:0, data:500
id:1, type:600, last:1, error:0, data:hello
id:0, type:0, last:0, error:0, data:
