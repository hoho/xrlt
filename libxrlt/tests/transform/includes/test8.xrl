<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">
    <xrl:response>
        <xrl:include>
            <xrl:href>/tmp</xrl:href>
            <xrl:type>text</xrl:type>
            <xrl:success>hooray</xrl:success>
            <xrl:failure>oops: <xrl:value-of select="/" /><xrl:status /></xrl:failure>
        </xrl:include>
    </xrl:response>
</xrl:requestsheet>
