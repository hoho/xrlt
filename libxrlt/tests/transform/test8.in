id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:1, type:600, last:1, error: 0, data:AWESOME
id:0, type:0, last:0, error: 0, data:
