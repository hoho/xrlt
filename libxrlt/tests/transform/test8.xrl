<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">
    <xrl:function name="func1">
        <xrl:param name="p2">val2</xrl:param>
        <xrl:param name="p1" select="111" />
        <xrl:param name="p3">val3</xrl:param>
        <xrl:param name="p4" />
        <xrl:param name="p5" select="123 + 234" />

        <love>peace</love>
        <a>|</a>
        <xrl:value-of select="$p1" />
        <a>|</a>
        <xrl:value-of select="$p2" />
        <a>|</a>
        <xrl:value-of select="$p3" />
        <a>|</a>
        <xrl:value-of select="$p4" />
        <a>|</a>
        <xrl:value-of select="$p5" />
    </xrl:function>

    <xrl:response>
        <xrl:apply name="func1">
            <xrl:with-param name="p4">alala</xrl:with-param>
            <xrl:with-param name="p1">olol|<xrl:value-of select="123 +35" />|</xrl:with-param>
        </xrl:apply>
    </xrl:response>
</xrl:requestsheet>
