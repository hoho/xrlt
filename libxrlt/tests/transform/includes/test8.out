XRLT_STATUS_SUBREQUEST
sr id: 1
sr method: GET
sr type: TEXT
sr url: /tmp
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_CHUNK
chunk: oops: hello500
XRLT_STATUS_DONE
